src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -11201,9 +11202,6 @@ void WaterThermalTankData::SizeTankForSupplySide(EnergyPlusData &state)
 
     static constexpr std::string_view RoutineName("SizeTankForSupplySide");
 
-    Real64 Tstart = 14.44;
```

Comment:
* [ ] TODO: should this return early?

```
if (!this->VolumeWasAutoSized && !this->MaxCapacityWasAutoSized) {
    return;
}
```

SizeStandAloneWaterHeater does avoid doing work

https://github.com/NREL/EnergyPlus/blob/3c8844fefaa68973f8f1d8369407eed0b37cfec2/src/EnergyPlus/WaterThermalTanks.cc#L11555